deployment/ansible/development/group_vars/all.sample.yml

```diff
@@ -49,6 +51,24 @@ interpreters:
     # Point to the ipaddress/domain name of InaSAFE Realtime processor
     ipaddress: *inasafe_django_host_ip
 
+# Facts needed for InaSAFE Realtime
+# This is to setup Realtime Hazard services
+inasafe_realtime:
+  # Use prebuilt docker will generate docker-compose services using prebuilt
+  # docker image, if set to yes. If set to no, then it will assume the service
+  # available locally on another docker-compose service, for example when
+  # debugging in another repo
+  use_prebuilt_docker: yes
+  # set host working dir to the location that will contains inasafe realtime
+  # hazards
+  host_working_dir: ./realtime-hazard
```

Comment:
Can we make a diagram about where is what?, this directory mapping sometimes makes me confuse to follow.

Reply:
Yeah, I agree. It is all in my head for now.